src/workloads/replication/startup/2_0_50GB.yml

```diff
@@ -0,0 +1,27 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: >
+  Loads the data for the heavy phase.
```

Comment:
does heavy phase mean a large amount of data ?

Reply:
yes.